hugo.yaml

```diff
@@ -38,8 +41,17 @@ params:
       [CC BY 4.0](https://creativecommons.org/licenses/by/4.0) |
     from_year: 2023
       #privacy_policy: link to privacy policy
-  version_menu: Releases
+
+  version_menu: Documentation 
+  version_menu_pagelinks: false
   url_latest_version: https://parodos.dev
+  versions:
+    - version: main
```

Comment:
why `main` and not `1.2-rc` ?

Reply:
I think `main` reflects the current work in progress, maybe `SNAPSHOT` would be better? Or do we want to stick as much as possible with the branch naming we have in the repo?